src/main/java/io/quarkus/gizmo2/Annotatable.java

```diff
@@ -22,8 +23,14 @@ public sealed interface Annotatable permits MemberCreator, ParamCreator, TypeCre
      * Add an annotation with no elements. If {@code annotationClass} has no {@link Retention}
      * annotation, {@link RetentionPolicy#RUNTIME} is assumed. This is the most commonly used
      * value, even though {@code CLASS} is the specified default.
+     * <p>
+     * If the annotation is {@linkplain Repeatable repeatable}, and adding this annotation would
+     * cause there to be more than one instance of the given annotation on this element,
+     * then the existing annotation and the given annotation are automatically wrapped in the
+     * repeatable annotation wrapper.
```

Comment:
```suggestion
     * container annotation.
```

it I believe the proper wording.